07-demystifying-reactivity.Rmd

```diff
@@ -278,38 +278,32 @@ attr(download_data, "spec") <- NULL
 str(download_data, 5)
 ```
 
-Now that the data is in memory, we will perform the **data manipulation**. The implementation details here are not that important; just understand that we're taking the `download_data` data frame from above, with its 3.4 million rows, and turning it into counts of rows grouped by the dimension in question (`package`).
+Now that the data is in memory, we will perform the **data manipulation**. The implementation details here are not that important; just understand that we're taking the `download_data` data frame from above, with its 3.4 million rows, and turning it into counts of rows grouped by the dimension in question (`package`), and arranging by the number of downloads.
 
 ```{r data_manip,cache=TRUE}
 ## Data manipulation =========
 
 # D. Count observations, grouping by the desired dimension
 by_frequency <- download_data %>%
-  pull(dimension) %>%
-  table() %>%
-  sort(decreasing = TRUE) %>%
-  as_tibble() %>%
-  setNames(c(dimension, "downloads"))
+  group_by_at(dimension) %>%
```

Comment:
should only need a simple `group_by` since not using vars/select semantics or additional functionality